README.md

```diff
@@ -56,7 +56,7 @@ route('events.venues.show', [event, venue]) // Returns '/events/1/venues/2'
 
 Practical AJAX example:
 
-```javascript
+```js
```

Comment:
Uniformity with [the rest of the README](https://raw.githubusercontent.com/tightenco/ziggy/master/README.md). All other JavaScript code snippets use `js`, except this one.